rocky/OOI_database_seed.json

```diff
@@ -476,9 +476,9 @@
     "fields": {
       "last_updated": "2021-08-09T14:50:09.835Z",
       "data": {
-        "description": "A system administration port is open.",
-        "recommendation": "Secure this port",
-        "risk": "Middle"
+        "description": "A port commonly used for system administration is open.",
+        "recommendation": "Manually validate whether this open port should actually be open.",
+        "risk": "Low"
```

Comment:
Open sysadmin ports risk should not be changed to low. This is a big security risk, but there are also a few usecases for having ssh open, so I think that keeping it middle instead of high is okay.